### DISCUSSION TITLE
Wo treffen wir uns am 16.Juli 24?

### DISCUSSION THREAD

**Discussion by scheffle** [scheffle/hamburg_audio_dev_stammtisch#2] [General]
Hi,
gibt es Vorschläge, wo wir uns das nächste Mal treffen wollen, oder sollen wir uns wieder im BrewDog treffen?

**Comment by georgco** [comment]
Moin! Ich wäre für jeden Vorschlag offen, es kann aber sonst auch gerne wieder das BrewDog werden. Wollen wir uns Ende Mai treffen oder diesmal ein größeres Intervall wählen, um die Sache spannend zu halten...? ;-)

**Comment by georgco** [comment]
Moin! Wie schon auf der Superbooth mit einigen besprochen, wollen wir das nächste Treffen kurz vor den HH-Sommerferien machen. Ich schlage daher mal den 16.07.24, 19 Uhr vor. Gibt es Vorschläge für den Ort?

**Reply by efluon** [reply]
Wie viele Leute sind zu erwarten?

**Reply by efluon** [reply]
Pappenheimer, oorlam, wald, malto wären bierfreundliche alternativen?

**Reply by efluon** [reply]
Mit mehr Platz fällt mit nur so was ein wie omas apotheke oder fritz bauch (gibs das noch?)

**Reply by scheffle** [reply]
Ich schätze mal es waren so um die 20 Leute beim ersten mal anwesend.

**Reply by scheffle** [reply]
Und wenn das Wetter mitmacht, wäre draussen natürlich schöner.

**Reply by efluon** [reply]
Das wird ja bestimmt nicht weniger. Aber ist natürlich eine Herausforderung für Aussenausschank. Und auch drinnen nur mit Reservierung machbar denk ich.

**Reply by scheffle** [reply]
Das Park Cafe bei mir ums Eck ginge wahrscheinlich. https://maps.app.goo.gl/K9BWLWqB1TtU4deDA

**Reply by georgco** [reply]
Draußen wäre natürlich schön. Vielleicht können wir es ja so machen, dass wir hier dann kurzfristig einen alternativen Ort vorschlagen, wenn  das Wetter nicht mitspielt.

**Reply by efluon** [reply]
Da ist auf jeden Fall schön informell, und Tischtennis oder Skaten geht auch noch :)

**Reply by okriwat-fender** [reply]
Das tanzende Einhorn wurde ja neu aufgebaut und hat jetzt auch relativ viel Platz draußen wie drinnen. 
Allerdings Mo/Di geschlossen, also vielleicht was für's übernächste mal :-)

[Zum tanzenden Einhorn - Google Maps](https://www.google.com/maps/place/Zum+tanzenden+Einhorn/@53.5606598,10.0521957,17z/data=!3m1!4b1!4m16!1m9!4m8!1m0!1m6!1m2!1s0x47b18f37af306dbf:0xaf28dd58248f5211!2sHammer+Steindamm+108,+20535+Hamburg!2m2!1d10.0547706!2d53.5606598!3m5!1s0x47b18f37af306dbf:0xaf28dd58248f5211!8m2!3d53.5606598!4d10.0547706!16s%2Fg%2F11kbg8qq1j?entry=ttu)

**Reply by thomaskvnze** [reply]
Hab mal hier subscribed, um den Ort mitzubekommen. Ich bin dabei!

**Reply by georgco** [reply]
Alles klar! Sieht ja jetzt so aus, dass wir uns im Park Cafe treffen, sofern das Wetter mitspielt.

**Reply by georgco** [reply]
Moin! Ich habe gerade gesehen, dass das Park Cafe laut Google nur bis 20 Uhr geöffnet hat. Stimmt das, Arne? Das wäre dann ja eher suboptimal...

**Reply by scheffle** [reply]
Moin, das sind die Winter Öffnungszeiten. Im Sommer schließen sie nach Wetter und Besuchern.

**Reply by georgco** [reply]
Alles klar. Dann planen wir mal fest gutes Wetter ein und treffen uns im Park Cafe, Unzerstraße 18A, 22767 Hamburg, am 16.07.24 um 19 Uhr.

Falls uns das Wetter doch einen Streich spielt, bestimmen wir an dieser Stelle kurzfristig einen neuen Ort.

Es wird großartig!

**Reply by georgco** [reply]
Moin! Mal eine Frage in die Runde: Wer wäre denn morgen alles mit dabei? Bislang gab es noch nicht so viele Reaktionen, bzw. eher Absagen. Sollten wir es vielleicht doch verschieben?

**Reply by efluon** [reply]
Ich würde kommen, allerdings wohl erst um ca 20oo. 
Ein anderes Mal aber auch.

**Reply by scheffle** [reply]
Also ich bin so ab 19 Uhr da. Wenn keiner kommt geh ich halt wieder nach hause, hab's ja nicht weit 😋

**Comment by georgco** [comment]
Genau, lasst uns einfach um 19 Uhr treffen. Es kommt, wer kommt. Ein paar Rückmeldungen habe ich jetzt doch noch bekommen.
